.circleci/config.yml

```diff
@@ -122,7 +122,7 @@ jobs:
               go_arch: arm64
     - checkout
     - restore_cache:
-        key: << parameters.executor >>_go.mod_{{ checksum "go.sum" }}_{{ checksum "mk/dependencies/deps.lock" }}_{{ checksum ".circleci/config.yml" }}
+        key: << parameters.executor >>_go.mod_{{ checksum "go.sum" }}_{{ checksum "mk/dependencies/deps.lock" }}_{{ checksum ".circleci/config.yml" }}_{{ checksum "mk/dev.mk" }}
```

Comment:
This is not required: https://github.com/kumahq/kuma/blob/master/tools/dev/install-dev-tools.sh#L21

Reply:
I'm trying to figure out why the caching is broken right now. `deps.lock` changes after executing `make dev/tools`

Reply:
See differing keys for save/restore in this job: https://app.circleci.com/pipelines/github/kumahq/kuma/20211/workflows/ddaa6ac0-0376-492e-864e-0f1d5cfe2ad6/jobs/345773